content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,255 @@
+
+To access the console, first, navigate to either Avalara's `sandbox
+<https://sandbox.admin.avalara.com/>`_ or `production <https://admin.avalara.com/>`_ environment.
+This will depend on which type of account was set in the :doc:`integration <../avatax>`. Log in to
+the management console.
+
+.. image:: avalara_portal/avalara_portal.png
+   :align: center
+   :alt: Avalara dashboard after logging into management portal.
+
+.. seealso::
+   For more information see Avalara's documentation: `Activate your Communications Customer Portal
+   account
+   <https://community.avalara.com/support/s/document-item?language=en_US&bundleId=qvv1656594440497&topicId=Activate_your_Communications_Customer_Portal_account.html&_LANG=enus>`_.
+
+.. _avalara_portal/transactions:
+
+Transactions
+============
+
+To access transactions, click in the :guilabel:`Transactions` link on the main dashboard upon
+logging into the :ref:`avatax_portal`. To manually access the *Transactions* page, while logged into
+the Avalara console, navigate to :menuselection:`Transactions --> Transactions`.
+
+.. image:: avalara_portal/avalara_transactions.png
+   :align: center
+   :alt: Avalara portal with the transactions shortcut highlighted.
+
+Edit transaction
+----------------
+
+Click into a transaction to reveal more details about the transaction. These include sections on
+:guilabel:`Invoice detail`, :guilabel:`Additional info`, and :guilabel:`Customer info`. Click
+:icon:`fa-pencil` :guilabel:`Edit document details` to make changes to the transaction.
+
+A :guilabel:`Discount` can be added to adjust the invoice. This is especially useful in cases where
+the transaction has already synced with Avalara / *AvaTax* and changes need to be made afterward.
+
+.. _avalara_portal/filter:
+
+Filter
+------
+
+Filter transactions by setting the :guilabel:`From` and :guilabel:`To` fields, and configuring other
+fields to filter by, including:
+
+ - :guilabel:`Document Status` : either of the following options, :guilabel:`All`,
+   :guilabel:`Voided`, :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Document Code` : either of the following options, :guilabel:`Exactly match`,
+   :guilabel:`Starts with`, or :guilabel:`Contains`.
+ - :guilabel:`Customer/Vendor Code`: this is the customer/vendor code in Odoo (e.g. Contact18).
+ - :guilabel:`Country` : this is the country this tax was calculated in; this is a text field.
+ - :guilabel:`Region` : this is the region of the country, this will vary based on the
+   :guilabel:`Country` selection.
+ - :guilabel:`Document Type` : either of the following selections, :guilabel:`All`, :guilabel:`Sales
+   Invoice`, :guilabel:`Purchase Invoice`, :guilabel:`Return Invoice`, :guilabel:`Inventory Transfer
+   Inbound Invoice`, :guilabel:`Inventory Transfer Outbound Invoice`, or :guilabel:`Customs
+   Invoice`.
+ - :guilabel:`Import ID` : represents the import ID of the document.
```

Comment:
```suggestion
 - :guilabel:`Document Status`: any of the following options, :guilabel:`All`,
   :guilabel:`Voided`, :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
 - :guilabel:`Document Code : any of the following options, :guilabel:`Exactly match`,
   :guilabel:`Starts with`, or :guilabel:`Contains`.
 - :guilabel:`Customer/Vendor Code`: the customer/vendor code in Odoo (e.g. `Contact18`).
 - :guilabel:`Country`: the country this tax was calculated in; this is a text field.
 - :guilabel:`Region`: the region of the country, which varies based on the
   :guilabel:`Country` selection.
 - :guilabel:`Document Type`: any of the following selections, :guilabel:`All`, :guilabel:`Sales
   Invoice`, :guilabel:`Purchase Invoice`, :guilabel:`Return Invoice`, :guilabel:`Inventory Transfer
   Inbound Invoice`, :guilabel:`Inventory Transfer Outbound Invoice`, or :guilabel:`Customs
   Invoice`.
 - :guilabel:`Import ID`: represents the import ID of the document.
```
I'm not certain I am seeing the Document Type and Import ID options on the Transactions page (given the log info provided in the task) -- might be worth double-checking they are there during your revisions -- just to be safe 👍

Reply:
Noted, this is accessed by +Filters, I've added this in.